apps/researcher/src/messages/nl/messages.json

```diff
@@ -43,7 +43,12 @@
     "heritageObject": "Erfgoedobject"
   },
   "PersonCard": {
-    "person": "Persoon"
+    "person": "Persoon",
+    "relatedPersons": "Hoeveelheid gerelateerde objecten Onbekend (niet geïmplementeerd)",
```

Comment:
Is this indeed about a number ('hoeveelheid')? Or simply: 'Onbekend (niet geïmplementeerd)'?

Reply:
Only 'Onbekend (niet geïmplementeerd)' will remove the context for the user. See image

![image](https://github.com/colonial-heritage/colonial-collections/assets/1481602/971a9cb0-b4b3-4cb9-8360-d1bb7eb3aa2d)

I could do 'gerelateerde objecten Onbekend (niet geïmplementeerd)'. But I do think this section is about the amount of related objects.

I do see the key is wrong. It should be `relatedObjects`.

Reply:
If the word 'hoeveelheid' is a problem I will change it to 'Gerelateerde objecten onbekend (niet geïmplementeerd)'

Reply:
Thinking about this some more. I will add the amount. This will be a static 0. This will represent the expected behaviour better. I will keep the '(niet geïmplementeerd)' part, so it's clear this is always zero.